lib/Bio/EnsEMBL/DataCheck/Pipeline/AdvisoryDataCheckSummary.pm

```diff
@@ -0,0 +1,74 @@
+=head1 LICENSE
```

Comment:
As discussed, please rename it to "DataCheckMailSummary" since this module is generic.